packages/docusaurus-plugin-content-docs/src/lastUpdate.ts

```diff
@@ -31,8 +36,16 @@ export async function getFileLastUpdate(
     if (err instanceof GitNotFoundError && !showedGitRequirementError) {
       logger.warn('Sorry, the docs plugin last update options require Git.');
       showedGitRequirementError = true;
+    } else if (
+      err instanceof FileNotTrackedError &&
+      !showedFileNotTrackedError
```

Comment:
not 100% sure to understand the goal of this condition here

the 1st time you print a custom message, and the next times you fallback to the else branch that shows the raw FileNotTrackedError ? That looks a bit weird to me.

Can you add a test to cover the intended behavior please? (like having 2 untracked git files?

Reply:
BTW not convinced that using global variables is the solution, but we can keep this for now

When building for i18n, those variables are not "reset" so a warning for the first i18n locale might not be shown for the 2nd i18n locale anymore

Reply:
Oops, logic mistake there😅 To be fixed later

I think i18n is fine, since there's no "context" in this error about which file is not tracked. Before this error was silently swallowed anyways

Reply:
I can't believe I left this pass. 😅

If you guys haven't started to fix it yet, I can do it.

Reply:
Yep, you can go ahead!